src/libs/ReportUtils.ts

```diff
@@ -1211,7 +1211,8 @@ function getDefaultWorkspaceAvatar(workspaceName?: string): React.FC<SvgProps> {
 
 function getWorkspaceAvatar(report: OnyxEntry<Report>): UserUtils.AvatarSource {
     const workspaceName = getPolicyName(report, false, allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${report?.policyID}`]);
-    return allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${report?.policyID}`]?.avatar ?? getDefaultWorkspaceAvatar(workspaceName);
+    const avatar = allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${report?.policyID}`]?.avatar;
+    return avatar ? avatar : getDefaultWorkspaceAvatar(workspaceName);
```

Comment:
```suggestion
    return avatar && avatar !== '' ? avatar : getDefaultWorkspaceAvatar(workspaceName);
```
This looks ugly but inevitable to avoid ts error.
Otherwise disable lint on this line

Reply:
Opted for following which I think is a little nicer 

```Javascript
const avatar = allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${report?.policyID}`]?.avatar ?? '';
return !isEmpty(avatar) ? avatar : getDefaultWorkspaceAvatar(workspaceName);
```

Reply:
We try to avoid lodash, underscore in TS

Reply:
https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#migration-guidelines

<img width="1051" alt="Screenshot 2023-12-14 at 3 46 05 AM" src="https://github.com/Expensify/App/assets/108292595/1a238b93-fe40-441d-ad11-9d1b20fa4e92">

But seems like `isEmpty` is already used in this file

Reply:
Hmm, I was also wondering whether `isEmpty` couldn't be replaced with `avatar !== ''` here (even after the latest change), but I'm only 90% sure this is equivalent. JavaScript's handling of emptiness/truthness scares me.